lib/runtime/life/instance.go

```diff
@@ -69,8 +71,25 @@ func NewRuntimeFromGenesis(g *genesis.Genesis, cfg *Config) (runtime.Instance, e
 	return NewInstance(code, cfg)
 }
 
+// NewInstanceFromFile instantiates a runtime from a .wasm file
+func NewInstanceFromFile(fp string, cfg *Config) (*Instance, error) {
+	// Reads the WebAssembly module as bytes.
+	// TODO determine lint issue: Potential file inclusion via variable
+	bytes, err := ioutil.ReadFile(fp) // nolint
```

Comment:
```suggestion
	bytes, err := ioutil.ReadFile(filepath.Clean(fp))
```

this fixes the lint issue

Reply:
Thanks, updated.